pkg/webhook/deploy/deployment.go

```diff
@@ -46,6 +53,51 @@ func Webhook(ctx context.Context, cl runtimeclient.Client, s *runtime.Scheme, na
 	return nil
 }
 
+// Delete deletes the webhook app if it's deployed. Does nothing if it's not.
+// Returns true if the app was deleted.
+func Delete(ctx context.Context, cl runtimeclient.Client, s *runtime.Scheme, namespace string, oldObjectOnly bool) (bool, error) {
+	objs, err := getTemplateObjects(s, namespace, "0", []byte{00000001})
+	if err != nil {
+		return false, err
+	}
+
+	var deleted bool
+	for _, obj := range objs {
+		unst := &unstructured.Unstructured{}
+		unst.SetGroupVersionKind(obj.GetObjectKind().GroupVersionKind())
+		objName := obj.GetName()
+		_, doNotDeleteFound := obj.GetLabels()["toolchain.dev.openshift.com/no-deletion"]
+		if doNotDeleteFound {
```

Comment:
I would really use an annotation for it - the purpose of annotations is to provide an additional context that can influence the execution/process of the resources. Label is used mainly for matching.
I believe that this is more closer to the first usage.

Reply:
Makes sense, annotation is better in this case. Thanks!

I've converted those in https://github.com/codeready-toolchain/member-operator/pull/562/commits/6f50ffab500165de806d03edc0b2b28807fd8d9c